src/core/ext/transport/binder/client/channel_create.cc

```diff
@@ -16,14 +16,23 @@
 
 #include "src/core/ext/transport/binder/client/channel_create.h"
 
-#if defined(ANDROID) || defined(__ANDROID__)
+// The interface is only defined if GPR_ANDROID is defined, because some
+// arguments requires JNI.
+// Furthermore, the interface is non-phony only when
+// GPR_SUPPORT_BINDER_TRANSPORT is true because actual implementation of binder
+// transport requires newer version of NDK API
+
+#ifdef GPR_ANDROID
+
+#include <grpc/grpc.h>
```

Comment:
Any reason we include these files even if GRP_SUPPORT_BINDER_TRANSPORT is false?

Reply:
Because `GRP_ASSERT(0)` is used below when GRP_SUPPORT_BINDER_TRANSPORT is false